### PR TITLE
Issue and PR templates

### LINKED ISSUE #54: Create templates for PRs and Issues
Making a default template for PRs and Issues according to GitHub documentation

### PR DESCRIPTION
**Pull request by AvivLiberman**
added the files into folders so we can add templates in the future

Fixed #54

### CODE REVIEW

**Review by orzionpour — APPROVED** [review]

**Comment by natifridman** [comment]
You can also add labels, assignees, etc automatically using a YAML format at the beginning of the file:
```
---
name: Tracking issue
about: Use this template for tracking new features.
title: "[DATE]: [FEATURE NAME]"
labels: tracking issue, needs triage
assignees: octocat
---
```

If you think it's relevant you can add some of this to the templates.

**Review by Asafh7 — APPROVED** [review]

**Comment by mgold1234** [comment]
LGTD

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+## Actual Behavior
+
+## Steps to Reproduce the Problem
+
+1.
+1.
+1.
+
+**Please mention somsone to take care of the issue**
```
Comment (kobihk) [inline .github/ISSUE_TEMPLATE/bug.md:11]:
```suggestion
**Please mention someone to take care of the issue**
```

File: .github/ISSUE_TEMPLATE/feature.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## What the feature does?
+
+1.
+1.
+1.
+
+**Please mention somsone to take care of the issue**
```
Comment (kobihk) [inline .github/ISSUE_TEMPLATE/feature.md:7]:
```suggestion
**Please mention someone to take care of the issue**
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+Fixes #
+
+## Changes you made
+
+-
+-
+-
+
+**Please mention somsone for code review**
```
Comment (kobihk) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:9]:
```suggestion
**Please mention someone for code review**
```

**Review by kobihk — CHANGES_REQUESTED** [review]

**Review by natifridman — APPROVED** [review]

**Review by kobihk — APPROVED** [review]

**Pull request merged**